README.rst

```diff
@@ -37,10 +37,19 @@ Configuration parameters:
 Generating Reports
 ==================
 
-You will need to install ``gnuplot`` as a system dependency. Running the following command will generate a report for the specified project in ``output`` directory.
+You will need to install ``gnuplot`` as a system dependency and the python dependencies. 
 
 .. code-block:: bash
 
+    $ brew install gnuplot # Mac OS
+    $ apt-get install gnuplot # Ubuntu etc.
+    $ virtualenv -p python3 venv
+    $ activate venv/bin/active
```

Comment:
I suggest removing the virtualenv part as it assumes user's knowledge and specific env setup, and it is not required if someone prefers installing dependencies systemwide.